src/app/screens/BookDetail/index.tsx

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import { useRoute } from '@react-navigation/native';
 import { BookDetailRouteType } from '@interfaces/routeType';
+import { ScrollView } from 'react-native-gesture-handler';
```

Comment:
⚠️ Sort imports, global imports first. Also => `import { ScrollView } from 'react-native';` not from `'react-native-gesture-handler'`